sites/next.skeleton.dev/src/examples/components/progress-rings/ExampleLabel.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+	import { ProgressRing } from '@skeletonlabs/skeleton-svelte';
+
+	let value = $state(25);
+	let max = 100;
+</script>
+
+<div class="grid grid-cols-1 gap-4 justify-center">
+	<ProgressRing {value} {max} showLabel />
+	<input type="range" class="input max-w-32" bind:value {max} />
```

Comment:
I think we go ahead and drop the range input for the label example. The focus should be the label itself. The first demo already shows you can bind the and manipulate the state like this.

Less code = easier for folks to digest!

Reply:
@phamduylong if you can implement this change I think we're good to go!